ui/app/templates/components/clients/running-total.hbs

```diff
@@ -9,8 +9,8 @@
       <div class="chart-header has-bottom-margin-xl">
         <h2 class="chart-title">Vault client counts</h2>
         <p class="chart-description">
-          A client is any user or service that interacts with Vault. They are made up of entity clients and non-entity
-          clients. The total client count number is an important consideration for Vault billing.
+          The total billable clients in the specified date range. This includes entity, non-entity, and secrets sync clients.
```

Comment:
I think we should remove `billable` here, the "billable clients" is something that's negotiated/deliberated which is why we mention below that it's "an important consideration"

Reply:
I took this copy from the latest design so I'm hesitant to remove if it's something that may have been deliberately added.

Reply:
I highly doubt it based on the conversations I've been a part of but I'll ask!